tests/test_scan.py

```diff
@@ -8,8 +8,11 @@
 import pytest
 from pyspark.sql import DataFrame, SparkSession
 from pyspark.sql import types as T  # noqa: N812
+from pyspark.sql.functions import col, when
```

Comment:
See a previous comment that might remove the need for this. If not, it is common practice to import pyspark functions as `F`, just like the `T` one line above. Then call `col` and `when` as `F.col` and `F.when`

Reply:
Updated to using functions as `F`